sha2/src/sha512/x86.rs

```diff
@@ -355,3 +355,64 @@ const SHA512_ROUNDS_NUM: usize = 80;
 const SHA512_HASH_BYTE_LEN: usize = 64;
 const SHA512_HASH_WORDS_NUM: usize = SHA512_HASH_BYTE_LEN / size_of::<u64>();
 const SHA512_BLOCK_WORDS_NUM: usize = SHA512_BLOCK_BYTE_LEN / size_of::<u64>();
+
+#[cfg(test)]
+mod tests {
+    use crate::sha512::x86::sha512_compress_x86_64_avx2;
+
+    #[test]
+    fn test_sha512_compress_x86_64_avx2() {
```

Comment:
This is not a correct way to test this module. You should check that SHA-NI is available, either at compile, or run time.

Reply:
Thanks for your time and I will manually resovle it.